### ISSUE TITLE
examples/IBFE/explicit/ex7 broken

### ISSUE THREAD

**Issue by deleeke** [IBAMR/IBAMR#181]
### Issue type
Bug

### Affected branches/commits
Branch: master
Commit: fdd3bd149cebbbd0d32681b31f217c99acafb585

### Description of problem
Looks like there is a compilation error in this example when compiling with GCC

I have not looked into it any further, but this looks like an undefined variable problem.

### Compilation error output:
```bash
In file included from test_main.cpp:2:0:
example.cpp: In function 'bool run_example(int, char**)':
example.cpp:227:72: error: 'boundary_id' was not declared in this scope
                     if (boundary_info->has_boundary_id(elem,side,1) || boundary_id == boundary_info->has_boundary_id(elem,side,3))
                                                                        ^
In file included from test_main.cpp:2:0:
example.cpp:255:72: error: 'boundary_id' was not declared in this scope
                     if (boundary_info->has_boundary_id(elem,side,1) || boundary_id == boundary_info->has_boundary_id(elem,side,3))
                                                                        ^
Makefile:612: recipe for target 'test2d-test_main.o' failed
```

### Steps to reproduce:
Configure IBAMR to build with `gcc` based compilers.
Attempt to compile the examples in examples/IBFE/explicit/ex7

**Comment by deleeke** [comment]
@raveenak96 

The Jenkins build of master is failing because of this. I filed this as an example of how you can let others know when the automated jobs catch errors like this.

**Comment by boyceg** [comment]
I think that there is a PR pending that fixes this. Will try to review it tonight...

**Comment by boyceg** [comment]
Fixed by #180.
